### PR TITLE
feat(NewOrderAheadApp/Menu Item Screen):Added Menu Item screen for new order ahead app.

### PR DESCRIPTION
**Pull request by dabralyogesh12**
Issues: fixes https://github.com/SleekOrg/tracker/issues/273
feat(NewOrderAheadApp/Menu Item Screen): Added Menu Item screen for the new order-ahead app.
Description: Added the layout for the menu item screen.
Verification: Tested locally.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ import AddedToCart from './features/stall/AddedToCart';
 import OrderSent from './features/stall/OrderSent';
 import { OrderStatus } from './features/stall/OrderStatus';
 import { OrderConfirmation } from './features/stall/OrderConfirmation';
+import { MenuItem } from './features/stall/MenuItem';
```
Comment (teamSleek) [inline src/App.tsx:13]: Show zero where it is zero.

Screen Shot 2021-04-12 at 3 17 03 PM

File: src/features/stall/MenuItem.tsx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        <StallBanner
+          bannerImage="/img/ItemName.png"
+          callingParent="MenuItem"
+          logoUrl="/img/ItemLogo.png"
+          itemName="Ultimate Breakfast Wrap"
+        />
+
+        <Grid container lg={12} xs={12} sm={12} className={classes.aboutGrid}>
+          <Grid item xs={11} lg={10} className={classes.itemText}>
+            <AboutItem details="Free range egg, Cumberland sausage, and beechwood smoked bacon with Heinz tomato ketchup & béchamel sauce in a barmarked tortilla wrap." />
```
Comment (teamSleek) [inline src/features/stall/MenuItem.tsx:94]: Font seems off

File: src/features/stall/MenuItem.tsx [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+            <Typography
+              variant="body1"
+              component="div"
+              roboto
+              style={{ fontWeight: 700 }}
+            >
+              <Currency quantity={10.0} currency="USD" /> -
+              <Currency quantity={20.0} currency="USD" />
+            </Typography>
+            <AddDelete />
```
Comment (teamSleek) [inline src/features/stall/MenuItem.tsx:106]: Alignment is off
Screen Shot 2021-04-12 at 3 22 29 PM

File: src/features/stall/MenuItem.tsx [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+export const MenuItem = () => {
+  const classes = useStyles();
+  return (
+    <div className={classes.root}>
+      <Grid container>
+        <StallBanner
+          bannerImage="/img/ItemName.png"
+          callingParent="MenuItem"
+          logoUrl="/img/ItemLogo.png"
+          itemName="Ultimate Breakfast Wrap"
```
Comment (teamSleek) [inline src/features/stall/MenuItem.tsx:89]: `Ultimate Breakfast Wrap` grid seems off. If Figma the name is in 2 lines and your demo shows it in 3 lines

File: src/features/stall/MenuItem.tsx [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+          </Grid>
+          <Grid item xs={11} lg={10} className={classes.priceGrid}>
+            <Typography
+              variant="body1"
+              component="div"
+              roboto
+              style={{ fontWeight: 700 }}
+            >
+              <Currency quantity={10.0} currency="USD" /> -
+              <Currency quantity={20.0} currency="USD" />
```
Comment (teamSleek) [inline src/features/stall/MenuItem.tsx:104]: missing space with ` -`

File: src/features/stall/MenuItem.tsx [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+            </Typography>
+            <AddDelete />
+          </Grid>
+        </Grid>
```
Comment (teamSleek) [inline src/features/stall/MenuItem.tsx:108]: Alignment off
Screen Shot 2021-04-12 at 3 27 25 PM

File: src/features/stall/MenuItem.tsx [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+          justify="center"
+          alignItems="center"
+          direction="column"
+        >
+          <Grid
+            container
+            xs={10}
+            lg={8}
+            justify="center"
+            direction="column"
```
Comment (teamSleek) [inline src/features/stall/MenuItem.tsx:123]: Add markers such as 
* `* Required`
* `minimum 2`
* `maximum 6`

File: src/features/stall/Variations.tsx [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+);
+
+export default function CheckboxList() {
+  const classes = useStyles();
+  const [checked, setChecked] = React.useState([0]);
+  const maxCheck = 2;
+  const handleToggle = (value: number) => () => {
+    const currentIndex = checked.indexOf(value);
+    const newChecked = [...checked];
+    console.log(newChecked);
```
Comment (teamSleek) [inline src/features/stall/Variations.tsx:28]: remove log or convert to debug

**Review by teamSleek — CHANGES_REQUESTED** [review]

File: src/features/stall/MenuItem.tsx (hunk 4 above)
Comment (dabralyogesh12) [reply src/features/stall/MenuItem.tsx:89]: Yes, that because of screen width.

File: src/App.tsx (hunk 1 above)
Comment (dabralyogesh12) [reply src/App.tsx:13]: Done, updated PR.

File: src/features/stall/MenuItem.tsx (hunk 2 above)
Comment (dabralyogesh12) [reply src/features/stall/MenuItem.tsx:94]: It's the same as in Figma.

File: src/features/stall/MenuItem.tsx (hunk 3 above)
Comment (dabralyogesh12) [reply src/features/stall/MenuItem.tsx:106]: Done, updated PR.

File: src/features/stall/MenuItem.tsx (hunk 5 above)
Comment (dabralyogesh12) [reply src/features/stall/MenuItem.tsx:104]: Done, updated PR.

File: src/features/stall/MenuItem.tsx (hunk 6 above)
Comment (dabralyogesh12) [reply src/features/stall/MenuItem.tsx:108]: Done, updated PR.

File: src/features/stall/MenuItem.tsx (hunk 7 above)
Comment (dabralyogesh12) [reply src/features/stall/MenuItem.tsx:123]: Will add that during integration.

File: src/features/stall/Variations.tsx (hunk 8 above)
Comment (dabralyogesh12) [reply src/features/stall/Variations.tsx]: Done, updated PR.

**Pull request merged**